admin/package-files/flocker-control-agent.firewalld.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Perhaps we should have separate folders for different OSes, instead of mixing CentOS and Ubuntu in the same folder?